src/SFA.DAS.ApprenticeCommitments.Web.MockServer/Program.cs

```diff
@@ -8,6 +8,7 @@ public static void Main(string[] args)
         {
             var apprenticeCommitmentsApi = ApprenticeCommitmentsApiBuilder.Create(5121)
                 .WithUsersFirstLogin()
+                .WithUsersCurrentApprenticeship()
```

Comment:
Not sure if we need a concept of currentApprenticeship here

Reply:
Renamed to match current API structure.